packages/jest-emotion/test/react-enzyme.test.js

```diff
@@ -1,130 +1,201 @@
-import 'test-utils/legacy-env'
+import 'test-utils/enzyme-env'
 /** @jsx jsx */
+
+import jestInCase from 'jest-in-case'
 import * as enzyme from 'enzyme'
-import { jsx } from '@emotion/core'
-import { createSerializer as createEnzymeSerializer } from 'enzyme-to-json'
-import { createSerializer } from 'jest-emotion'
-import { toMatchSnapshot } from 'jest-snapshot'
+import { jsx, ThemeProvider } from '@emotion/core'
+import styled from '@emotion/styled'
 import React from 'react'
+import toJson from 'enzyme-to-json'
 
-const createEnzymeSnapshotMatcher = serializerOptions => {
-  const serializer = createEnzymeSerializer(serializerOptions)
-  const identityPrinter = v => v
-
-  return function(val) {
-    return toMatchSnapshot.call(this, serializer.print(val, identityPrinter))
-  }
-}
+import serializer from '../src/enzyme'
```

Comment:
We want to have imports like this so we can run tests against the built bundles
```suggestion
import serializer from 'jest-emotion/enzyme'
```

Reply:
thought i was having problems when it was that way, but this works, thanks for the fix